packages/migration/src/migrations/events/1756312338964_add_status_for_locations.sql

```diff
@@ -0,0 +1,5 @@
+-- Up Migration
+ALTER TABLE locations ADD COLUMN status text CHECK (status IN ('active', 'inactive')) DEFAULT 'active';
```

Comment:
we have generally created status enums
```
CREATE TYPE action_status AS ENUM ('Requested', 'Accepted', 'Rejected');
```

or `inactivated_at` 

should we consider either here